/**:
  ros__parameters:
    max_speed: 2.0  # Maximum speed in m/s
    max_steering_angle: 0.34  # Maximum steering angle in radians
    safety_distance: 0.5  # Distance to start slowing down in meters
    stop_distance: 0.3  # Distance to completely stop in meters
    angle_increment: 0.0058  # LiDAR angle increment in radians 